pkg/render/monitor/monitor.go

```diff
@@ -684,6 +684,9 @@ func (mc *monitorComponent) prometheusServiceService() *corev1.Service {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      PrometheusServiceServiceName,
 			Namespace: common.TigeraPrometheusNamespace,
+			Labels: map[string]string{
+				"k8s-app": TigeraPrometheusObjectName,
```

Comment:
Does this need to be `k8s-app`? Or is this just one we've chosen?
I'm hesitant to use this since it is a standard k8s label for pods, is it also a standard one for Services?

Reply:
Looks like we use `k8s-app` for other service labels already.